src/game/savegame.c

```diff
@@ -445,12 +452,13 @@ void Savegame_Shutdown(void)
 
 void Savegame_ScanSavedGames(void)
 {
+    Savegame_Init();
```

Comment:
Please flip the order of these calls as currently they make little semantic sense (init and immediately clean up)?